server/src/main/scala/controllers/package.scala

```diff
@@ -58,4 +43,22 @@ package object controllers {
       logger.debug(s"Error response while handling a request: ${ex.getMessage}", ex)
       Left(ErrorResponse(ex.getMessage))
   }
+
+  // UUID has to be future, because we want to handle the exception in the controllers
+  implicit def authHandler(serverRequest: ServerRequest)(implicit ec: ExecutionContext): Future[UUID] =
+    val session = serverRequest.underlying
```

Comment:
Let's comment `This is the way to access the play request from tapir, we need it to extract the play session`